testapps/iOSImageFormatProfilingTestApp/test/AppDelegate.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+	var window: UIWindow?
+
+
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+		// Override point for customization after application launch.
```

Comment:
Can we just remove most (all?) of the functions in this file since we don't actually override any of them?